RedHatInsights/insights-inventory-frontend
# fix(Groups): Fix incorrect API keys resolution

(no jira)

This polishes some of the components and ensures tables read right data from responses according to https://console.stage.redhat.com/docs/api/inventory/v1.

## How to test

1. Navigate to /groups with `insights-qa`
2. Make sure all data is rendered correctly: columns have valid data. Verify sorting, filtering, pagination.
3. Go to any group with/without hosts, make sure there are no new visible bugs, data is rendered correctly, if group has hosts, then all these hosts are shown in the table.
